src/components/Icon/Icon.scss

```diff
@@ -66,6 +66,14 @@
   }
 }
 
+.colorInteractive {
+  @include recolor-icon(var(--p-interactive));
+
+  &::before {
+    background-color: var(--p-surface-selected);
+  }
```

Comment:
We can 🔥 this unless you need it for some reason. We're trying to do away with the backdrop prop on Icon which is what this `before` background is for. If you do need this for whatever reason we should add a specific color in polaris-tokens based on interactive vs surface

Reply:
No problem, either way. 😊 I was matching the Figma design that has the `--p-surface-selected` as the arrow color, but it's a subtle difference.

Reply:
Ah, I see the layer is set to pass through in Figma and isn't needed. Removed – all set! 🔥